test/units/modules/cloud/amazon/test_redshift_iam_roles.py

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Are you able to add some integration tests for this module as well?
https://github.com/ansible/community/blob/master/group-aws/integration.md#policy

Reply:
I've been taking a stab at this over the past couple days and it looks like I'm running into issues that will remedied with https://github.com/ansible/ansible/pull/37052 (and described in https://github.com/ansible/ansible/issues/37051).  I'll be watching progress on both.